[Bug]: v0.6.2 引入代理环境变量支持后导致 WebSocket 连接 127.0.0.1 失败

### 部署平台

Docker

### 版本号

v0.6.2

### 客户端/工具

1Panel

### 请求接口

None

### 问题/请求描述

在升级到v0.6.2后，如果环境变量中配置了HTTP_PROXY 或HTTPS_PROXY，Camoufox会尝试通过代理连接 ws://127.0.0.1:9998，导致Connection Refused，加入NO_PROXY=localhost,127.0.0.1,0.0.0.0后正常

### 日志信息（可选）

```text
[INFO] 2026-02-12 14:15:50.567 [Asia/Shanghai] [ProxySystem] - [System] Loaded "immediate switch status codes" from default value.

[INFO] 2026-02-12 14:15:50.823 [Asia/Shanghai] [ProxySystem] - [System] Successfully loaded 24 models from models.json.

[INFO] 2026-02-12 14:15:50.826 [Asia/Shanghai] [ProxySystem] - ================ [ Active Configuration ] ================

[INFO] 2026-02-12 14:15:50.827 [Asia/Shanghai] [ProxySystem] - HTTP Server Port: 7860

[INFO] 2026-02-12 14:15:50.829 [Asia/Shanghai] [ProxySystem] - Listening Address: 0.0.0.0

[INFO] 2026-02-12 14:15:50.831 [Asia/Shanghai] [ProxySystem] - Streaming Mode: real

[INFO] 2026-02-12 14:15:50.833 [Asia/Shanghai] [ProxySystem] - Force Thinking: false

[INFO] 2026-02-12 14:15:50.835 [Asia/Shanghai] [ProxySystem] - Force Web Search: false

[INFO] 2026-02-12 14:15:50.837 [Asia/Shanghai] [ProxySystem] - Force URL Context: false

[INFO] 2026-02-12 14:15:50.839 [Asia/Shanghai] [ProxySystem] - Auto Update Auth: true

[INFO] 2026-02-12 14:15:50.841 [Asia/Shanghai] [ProxySystem] - Usage-based Switch Threshold: Disabled

[INFO] 2026-02-12 14:15:50.843 [Asia/Shanghai] [ProxySystem] - Failure-based Switch: Switch after 3 failures

[INFO] 2026-02-12 14:15:50.844 [Asia/Shanghai] [ProxySystem] - Immediate Switch Status Codes: 429, 503

[INFO] 2026-02-12 14:15:50.846 [Asia/Shanghai] [ProxySystem] - Max Retries per Request: 3 times

[INFO] 2026-02-12 14:15:50.848 [Asia/Shanghai] [ProxySystem] - Retry Delay: 2000ms

[INFO] 2026-02-12 14:15:50.849 [Asia/Shanghai] [ProxySystem] - API Key Source: Custom

[INFO] 2026-02-12 14:15:50.851 [Asia/Shanghai] [ProxySystem] - =============================================================

[INFO] 2026-02-12 14:15:50.853 [Asia/Shanghai] [ProxySystem] - [Auth] Using files in "configs/auth/" directory for authentication.

[INFO] 2026-02-12 14:15:50.859 [Asia/Shanghai] [ProxySystem] - [Auth] Auth file scan detected changes. Reloading and re-validating...

[INFO] 2026-02-12 14:15:50.867 [Asia/Shanghai] [ProxySystem] - [Auth] Reload complete. 2 valid sources available: [0, 1]

[INFO] 2026-02-12 14:15:50.873 [Asia/Shanghai] [ProxySystem] - [Auth] Rate limiting enabled: 5 attempts per 15 minutes

[INFO] 2026-02-12 14:15:50.876 [Asia/Shanghai] [ProxySystem] - [System] Starting flexible startup process...

[INFO] 2026-02-12 14:15:50.974 [Asia/Shanghai] [ProxySystem] - [System] HTTP server is listening on http://0.0.0.0:7860

[INFO] 2026-02-12 14:15:50.976 [Asia/Shanghai] [ProxySystem] - [System] Keep-Alive timeout set to 120 seconds.

[INFO] 2026-02-12 14:15:50.986 [Asia/Shanghai] [ProxySystem] - [System] Proxy server system startup complete.

[INFO] 2026-02-12 14:15:50.988 [Asia/Shanghai] [ProxySystem] - [System] No valid startup index specified, will try in default order [0, 1].

[INFO] 2026-02-12 14:15:50.990 [Asia/Shanghai] [ProxySystem] - [System] Attempting to start service with account #0...

[INFO] 2026-02-12 14:15:50.995 [Asia/Shanghai] [ProxySystem] - [Browser] 🌐 Using proxy: http://*.*.*.*:7890

[INFO] 2026-02-12 14:15:50.996 [Asia/Shanghai] [ProxySystem] - 🚀 [Browser] Main browser instance not running, performing first-time launch...

[INFO] 2026-02-12 14:15:53.251 [Asia/Shanghai] [ProxySystem] - [Auth] Successful login from IP: *.*.*.*

[INFO] 2026-02-12 14:15:53.529 [Asia/Shanghai] [ProxySystem] - [Entrypoint] Received a request: GET /api/version/check

[INFO] 2026-02-12 14:16:01.326 [Asia/Shanghai] [ProxySystem] - ✅ [Browser] Main browser instance successfully launched.

[INFO] 2026-02-12 14:16:01.328 [Asia/Shanghai] [ProxySystem] - ==================================================

[INFO] 2026-02-12 14:16:01.329 [Asia/Shanghai] [ProxySystem] - 🔄 [Browser] Creating new API browser context for account #0

[INFO] 2026-02-12 14:16:01.331 [Asia/Shanghai] [ProxySystem] - • Auth source: File auth-0.json

[INFO] 2026-02-12 14:16:01.333 [Asia/Shanghai] [ProxySystem] - ==================================================

[INFO] 2026-02-12 14:16:01.346 [Asia/Shanghai] [ProxySystem] - [Config] Found port config line: constructor(endpoint = "ws://127.0.0.1:9998") {

[INFO] 2026-02-12 14:16:01.348 [Asia/Shanghai] [ProxySystem] - [Config] Replaced with: constructor(endpoint = "ws://127.0.0.1:9998") {

[INFO] 2026-02-12 14:16:13.523 [Asia/Shanghai] [ProxySystem] - [Browser] ⚡ Forced window wake-up via JS focus.

[INFO] 2026-02-12 14:16:13.528 [Asia/Shanghai] [ProxySystem] - [Browser] Navigating to target page...

[INFO] 2026-02-12 14:16:16.738 [Asia/Shanghai] [ProxySystem] - [Browser] Privacy protection layer active: ANGLE (NVIDIA, NVIDIA GeForce GTX 1050 Ti Direct3D11 vs_5_0 ps_5_0, D3D11)

[INFO] 2026-02-12 14:16:16.856 [Asia/Shanghai] [ProxySystem] - [Browser] Privacy protection layer active: ANGLE (NVIDIA, NVIDIA GeForce GTX 1050 Ti Direct3D11 vs_5_0 ps_5_0, D3D11)

[INFO] 2026-02-12 14:16:22.276 [Asia/Shanghai] [ProxySystem] - [Browser] Page loaded.

[INFO] 2026-02-12 14:16:32.821 [Asia/Shanghai] [ProxySystem] - [Browser] ✅ Executed realistic page activation (Random -> 1,1 Click).

[INFO] 2026-02-12 14:16:35.902 [Asia/Shanghai] [ProxySystem] - [Browser] [Diagnostic] URL: https://aistudio.google.com/u/0/apps/bundled/blank?showPreview=true&showAssistant=true

[INFO] 2026-02-12 14:16:35.904 [Asia/Shanghai] [ProxySystem] - [Browser] [Diagnostic] Title: "Empty | Google AI Studio"

[INFO] 2026-02-12 14:16:35.907 [Asia/Shanghai] [ProxySystem] - [Browser] 🔍 Starting intelligent popup detection (max 6s)...

[INFO] 2026-02-12 14:17:03.858 [Asia/Shanghai] [ProxySystem] - [Browser] ✅ Popup detection complete (6 iterations, 0 popups handled)

[INFO] 2026-02-12 14:17:03.864 [Asia/Shanghai] [ProxySystem] - [Browser] Preparing UI interaction, forcefully removing all possible overlay layers...

[INFO] 2026-02-12 14:17:04.014 [Asia/Shanghai] [ProxySystem] - [Browser] (Step 1/5) Preparing to click "Code" button...

[INFO] 2026-02-12 14:17:04.019 [Asia/Shanghai] [ProxySystem] - [Attempt 1/15] Cleaning overlay layers and clicking...

[INFO] 2026-02-12 14:17:05.492 [Asia/Shanghai] [ProxySystem] - [Browser] Trying to locate "Code" entry point using smart selectors...

[INFO] 2026-02-12 14:17:05.970 [Asia/Shanghai] [ProxySystem] - [Browser] ✅ Smart match: "button:text("Code")", clicking...

[INFO] 2026-02-12 14:17:09.291 [Asia/Shanghai] [ProxySystem] - ✅ Click successful!

[INFO] 2026-02-12 14:17:09.294 [Asia/Shanghai] [ProxySystem] - [Browser] (Step 2/5) "Code" button clicked successfully, waiting for editor to become visible...

[INFO] 2026-02-12 14:17:11.401 [Asia/Shanghai] [ProxySystem] - [Browser] (Cleanup #2) Preparing to click editor, forcefully removing all possible overlay layers again...

[INFO] 2026-02-12 14:17:11.674 [Asia/Shanghai] [ProxySystem] - [Browser] (Step 3/5) Editor displayed, focusing and pasting script...

[INFO] 2026-02-12 14:17:15.226 [Asia/Shanghai] [ProxySystem] - [Browser] (Step 4/5) Script pasted.

[INFO] 2026-02-12 14:17:15.229 [Asia/Shanghai] [ProxySystem] - [Browser] (Step 5/5) Clicking "Preview" button to activate script...

[INFO] 2026-02-12 14:17:22.406 [Asia/Shanghai] [ProxySystem] - [Browser] ✅ UI interaction complete, script is now running.

[INFO] 2026-02-12 14:17:22.410 [Asia/Shanghai] [ProxySystem] - [Browser] ⚡ Sending active trigger request to Launch flow...

[INFO] 2026-02-12 14:17:22.990 [Asia/Shanghai] [ProxySystem] - [Browser] 🛡️ Background health monitor service (Scavenger) started...

[INFO] 2026-02-12 14:17:24.500 [Asia/Shanghai] [ProxySystem] - [Browser] 🛡️ Background Wakeup Service (Rocket Handler) started...

[INFO] 2026-02-12 14:17:25.868 [Asia/Shanghai] [ProxySystem] - [Browser] System initializing...

[INFO] 2026-02-12 14:17:25.875 [Asia/Shanghai] [ProxySystem] - [Browser] Connecting to server: ws://127.0.0.1:9998

[INFO] 2026-02-12 14:17:27.558 [Asia/Shanghai] [ProxySystem] - [Auth Update] 💾 Successfully updated auth credentials for account #0

[INFO] 2026-02-12 14:17:27.561 [Asia/Shanghai] [ProxySystem] - ==================================================

[INFO] 2026-02-12 14:17:27.563 [Asia/Shanghai] [ProxySystem] - ✅ [Browser] Account 0 context initialized successfully!

[INFO] 2026-02-12 14:17:27.565 [Asia/Shanghai] [ProxySystem] - ✅ [Browser] Browser client is ready.

[INFO] 2026-02-12 14:17:27.569 [Asia/Shanghai] [ProxySystem] - ==================================================

[INFO] 2026-02-12 14:17:27.572 [Asia/Shanghai] [ProxySystem] - [System] ✅ Successfully started with account #0!

[ERROR] 2026-02-12 14:17:28.792 [Asia/Shanghai] [ProxySystem] - [Browser Page Error] [JavaScript Error: "Camoufox can’t establish a connection to the server at ws://127.0.0.1:9998/." {file: "blob:https://4bub4uor6w4pdu09lmbekx7zvne7hx07ezdyv2uci2kvvaixq7-h868144788.scf.usercontent.goog/de77e031-5605-40f8-b067-4b8137d30c10" line: 287}]

[INFO] 2026-02-12 14:17:28.800 [Asia/Shanghai] [ProxySystem] - [Browser] WebSocket connection error: JSHandle@object

[INFO] 2026-02-12 14:17:28.805 [Asia/Shanghai] [ProxySystem] - [Browser] System initialization failed: undefined

[ERROR] 2026-02-12 14:17:28.815 [Asia/Shanghai] [ProxySystem] - [Browser Page Error] Proxy system startup failed: JSHandle@object

[INFO] 2026-02-12 14:17:28.820 [Asia/Shanghai] [ProxySystem] - [Browser] Proxy system startup failed: undefined

[INFO] 2026-02-12 14:17:28.826 [Asia/Shanghai] [ProxySystem] - [Browser] ❌ Connection disconnected, preparing to reconnect...
```

--- a/src/utils/ProxyUtils.js
+++ b/src/utils/ProxyUtils.js
@@ -38,11 +38,7 @@ const parseProxyFromEnv = () => {
             .map(s => s.trim())
             .filter(Boolean);
         finalBypass = [...new Set([...defaultBypass, ...userBypass])];
-    } else {
-        // If user didn't specify NO_PROXY, we should at least bypass local addresses
-        // to prevent WebSocket connection failures when proxy is set.
     }
-
     const bypassString = finalBypass.join(",");
 
     // Playwright expects: { server, bypass?, username?, password? }

